reconciletags/reconciletags.py

```diff
@@ -50,15 +50,17 @@ def reconcile_tags(self, data, dry_run):
         # want to see config regardless of whether we actually run the
         # reconciler.
         self.call('gcloud config list', False)
-        for repo, images in data.items():
-            logging.debug(self.get_existing_tags(repo))
+        for project in data['projects']:
+            for registry in project['registries']:
+                full_repo = registry + '/' + project['repository']
```

Comment:
Could you use os.path.join here? It would make it a little safer in case someone includes a / at the end of registry by accident.